bdi_game/src/game/game.rs

```diff
@@ -1,46 +1,52 @@
 use std::time::{Duration, Instant};
 
 use crate::{
-    display::game_display::GameDisplay,
+    display::sdl::*,
     input::Input,
     simulation::{simulation::Simulation, world_state::WorldState},
 };
 
 use super::logging::init_logging;
 
-pub struct Game<D: GameDisplay> {
-    display: D,
```

Comment:
Please don't, this trait here is for a reason

Reply:
It was quick fix to make sure examples are compiling because we had name clash with my/your display mod. I'll resolve it